Emby.Server.Implementations/Services/ServicePath.cs

```diff
@@ -403,46 +380,39 @@ public bool IsMatch(string httpMethod, string[] withPathInfoParts, out int wildc
                         continue;
                     }
 
-                    if (withPathInfoParts.Length <= pathIx || !LiteralsEqual(withPathInfoParts[pathIx], literalToMatch))
+                    if (withPathInfoParts.Length <= pathIx
+                        || !string.Equals(withPathInfoParts[pathIx], literalToMatch, StringComparison.InvariantCultureIgnoreCase))
                     {
                         return false;
                     }
+
                     pathIx++;
                 }
             }
 
             return pathIx == withPathInfoParts.Length;
         }
 
-        private static bool LiteralsEqual(string str1, string str2)
-        {
-            // Most cases
-            if (string.Equals(str1, str2, StringComparison.OrdinalIgnoreCase))
-            {
-                return true;
-            }
-
-            // Handle turkish i
```

Comment:
are you sure current case of using `StringComparison.InvariantCultureIgnoreCase` covers "turkish i" case?

Reply:
Afaik no, but the current implementation doesn't either (at least in my tests).